TestAssignmentDoItSoftware/TestAssignmentDoItSoftware/AuthScreen/AuthScreenViewController.h

```diff
@@ -0,0 +1,44 @@
+//
+//  AuthScreenViewController.h
+//  TestAssignmentDoItSoftware
+//
+//  Created by Vladimir Samoylenko on 2/15/19.
+//  Copyright © 2019 TheRedQueen. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+#import "MyTaskListScreenViewController.h"
+#import "TASingleton.h"
+
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface AuthScreenViewController : UIViewController<UITextFieldDelegate>{
+    
```

Comment:
Here and everywhere else there's no concise style applied to the source. While it's common to use one of the popular style guides (e.g. the [Google](http://google.github.io/styleguide/objcguide.html)'s one or [NYT](https://github.com/NYTimes/objective-c-style-guide)'s) the actual problem is consistency: same semantic constructions might be styled in different ways (in re. spaces and newlines) throughout this codebase.